File: Python/uri_1041.py
x, y = input().split(' ')
x = float(x)
y = float(y)
if x > 0 and y > 0:
    print('Q1')
elif x < 0 < y:
    print('Q2')
elif x < 0 and y < 0:
    print('Q3')
elif x > 0 > y:
    print('Q4')
elif x == 0 and (y > 0 or y < 0):
    print('Eixo Y')
elif y == 0 and (x > 0 or x < 0):
    print('Eixo X')
elif x == 0 and y == 0:
    print('Origem')

File: Go/uri_1041.go
package main

import "fmt"

func main() {
	var x, y float64
	fmt.Scan(&x, &y)
	if x > 0 && y > 0 {
		fmt.Print("Q1\n")
	} else if x < 0 && y > 0 {
		fmt.Print("Q2\n")
	} else if x < 0 && y < 0 {
		fmt.Print("Q3\n")
	} else if x > 0 && y < 0 {
		fmt.Print("Q4\n")
	} else {
		if x != 0 && y == 0 {
			fmt.Print("Eixo X\n")
		} else if x == 0 && y != 0 {
			fmt.Print("Eixo Y\n")
		} else {
			fmt.Print("Origem\n")
		}
	}
}
